cli/state.rs

```diff
@@ -65,6 +66,7 @@ pub struct State {
   pub start_time: Instant,
   pub resource: resources::Resource,
   pub dispatch_selector: ops::OpSelector,
+  pub modules: deno::Modules,
```

Comment:
This is something we need to do. However it needs to also be removed from the Worker struct. I will prepare a PR which does just this.

Reply:
`Modules` had been moved to `State` - @fewf you can merge latest changes from `master`